File: src/FrontEnd/NgClient/src/app/shared/models/anamnesis.ts
export interface IAnamnesis {
    referralId: string;
    categoryId: AnamnesisCategory;
    summary: string;
}

export enum AnamnesisCategory {
    "Анамнез жизни" = 1,
    "Анамнез заболевания" = 2,
    "Объективный статус" = 3
}

File: src/Services/Anamnesies/CORE/Anamnesies.Domain/Anamnesis.cs
using BaseDomain;

namespace Anamnesies.Domain
{
    /// <summary>
    /// Anamnesis of the disease
    /// </summary>
    public class Anamnesis : BaseDomainEntity
    {
        public string ReferralId { get; set; }
        public AnamnesisCategory CategoryId { get; set; }
        /// <summary>
        /// general information about the patient
        /// </summary>
        public string Summary { get; set; }
    }
}
